LZW/LZW/LZWcompressor.cs

```diff
@@ -0,0 +1,121 @@
+﻿// <copyright file="LZWcompressor.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace LZW;
+
+/// <summary>
+/// LZW algo with methods: Compress, Encode, WriteInFile.
+/// </summary>
+public class LZWcompressor
+{
+    /// <summary>
+    /// Forward conversion of LZW.
+    /// </summary>
+    /// <param name="sequence">Input array of bytes.</param>
+    /// <returns>Transformed array of int codes.</returns>
+    public static int[] Compress(byte[] sequence)
+    {
+        int code = 256;
+        var trieOfBytes = CompressionTrie.InitializeTrieBytes();
+
+        List<int> compressedSequence = new();
+        List<byte> currentElement = new();
+
+        foreach (byte element in sequence)
+        {
+            List<byte> tmpElement = new List<byte>(currentElement);
+            tmpElement.Add(element);
+
+            if (!trieOfBytes.Contains(tmpElement))
+            {
+                compressedSequence.Add(trieOfBytes.GetCode(currentElement).Code);
+                trieOfBytes.Add(tmpElement, code);
+                currentElement.Clear();
+                currentElement.Add(element);
+                code++;
+            }
+            else
+            {
+                currentElement = tmpElement;
+            }
+        }
+
+        if (currentElement.Count > 0)
+        {
+            compressedSequence.Add(trieOfBytes.GetCode(currentElement).Code);
+        }
+
+        return compressedSequence.ToArray();
+    }
+
+    /// <summary>
+    /// Transform sequence of int codes to variable length bit sequence.
+    /// </summary>
+    /// <param name="sequence">Array of int codes.</param>
+    /// <returns>Array of bytes.</returns>
+    public static byte[] EncodeByteSequencee(int[] sequence)
+    {
+        List<byte> result = new List<byte>();
```

Comment:
```suggestion
        List<byte> result = [];
```
Можно так или  ещё ``= new();``